sway-fmt-v2/src/fmt.rs

```diff
@@ -6,6 +6,7 @@ pub use crate::{
     config::manifest::Config,
     error::{ConfigError, FormatterError},
 };
+use crate::{config::whitespace::NewlineStyle, utils::newline_style::apply_newline_style};
```

Comment:
There's already a set of crate imports above - we should merge these imports into that set.

Reply:
I was also going to say something about this, but the above is public, I don't think these need to be necessarily since they just get used in a single method here, but I digress haha

Reply:
Ah good spot! My apologies, kaya disregard my comment :)

In general, it's good to try and be mindful of the public API and to take the more conservative choice when we're unsure (i.e. private, or `pub(crate)`). Technically, anything that changes in the public API is a breaking semver change, so if we're not careful it can make it trickier to reach a level of "stability".